appyx-interactions/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id("com.android.library")
+    id("kotlin-android")
```

Comment:
Is this a library or a sample? is it going to be published?

Reply:
Library. Eventually published, not for a while yet.